src/vivarium/framework/population/manager.py

```diff
@@ -284,22 +287,54 @@ def register_simulant_initializer(
         requires_streams
             The randomness streams necessary to initialize the simulant
             attributes.
+        required_resources
+            The resources that the initializer requires to run. Strings are
+            interpreted as column names, and Pipelines and RandomnessStreams
```

Comment:
This confused me for a second. Either

"Strings are interpreted as column names and Pipelines and RandomnessStreams are interpreted as value pipelines and randomness streams, respectively."

or

"Strings are interpreted as column names, Pipelines are interpreted as value pipelines, and RandomnessStreams are interpreted as randomness streams."